ETTERCAPP INSTALL ERROR

hi, 
ive done this to install ettercap as you commend in other issues:
git clone https://github.com/Ettercap/ettercap
cd ettercap
mkdir build
cd build
cmake -DCMAKE_BUILD_TYPE=Debug ..
make
sudo make install

*but when is finishing appears a message as below:

CMake Error at share/cmake_install.cmake:49 (file):
  file INSTALL cannot find
  "/home/tiago/ettercap/build/ettercap/build/share/etter.mdns": No such file
  or directory.
Call Stack (most recent call first):
  cmake_install.cmake:46 (include)

make: *** [Makefile:74: install] Error 1

CMake Error at share/cmake_install.cmake:49 (file):
  file INSTALL cannot find "/home/tiago/ettercap/build/share/etter.mdns": No
  such file or directory.
Call Stack (most recent call first):
  cmake_install.cmake:46 (include)


make: *** [Makefile:74: install] Error 1
root@Kali:/home/tiago/ettercap/build# pwd
/home/tiago/ettercap/build
root@Kali:/home/tiago/ettercap/build# ls -lh share/
total 1,1M
drwxr-xr-x 2 root root 4,0K jul  4 17:08 CMakeFiles
-rw-r--r-- 1 root root 5,1K jul  4 12:01 cmake_install.cmake
-rw-r--r-- 1 root root  60K jul  4 12:01 ettercap.png
-rw-r--r-- 1 root root  17K jul  4 12:01 ettercap-small.png
-rw-r--r-- 1 root root 6,6K jul  4 12:01 ettercap.svg
-rw-r--r-- 1 root root 8,7K jul  4 12:01 etter.conf
-rw-r--r-- 1 root root 4,4K jul  4 12:01 etter.dns
-rw-r--r-- 1 root root 1,8K jul  4 12:01 etter.dns.examples
-rw-r--r-- 1 root root 1,9K jul  4 12:01 etter.fields
-rw-r--r-- 1 root root 1,3K jul  4 12:01 etter.filter
-rw-r--r-- 1 root root 1,4K jul  4 12:01 etterfilter.cnt
-rw-r--r-- 1 root root 3,6K jul  4 12:01 etter.filter.examples
-rw-r--r-- 1 root root 1,3K jul  4 12:01 etter.filter.kill
-rw-r--r-- 1 root root 1,6K jul  4 12:01 etter.filter.pcre
-rw-r--r-- 1 root root 1,8K jul  4 12:01 etter.filter.ssh
-rw-r--r-- 1 root root 3,7K jul  4 12:01 etterfilter.tbl
-rw-r--r-- 1 root root 708K jul  4 12:01 etter.finger.mac
-rw-r--r-- 1 root root  85K jul  4 12:01 etter.finger.os
-rw-r--r-- 1 root root  940 jul  4 12:01 etterlog.dtd
-rw-r--r-- 1 root root  771 jul  4 12:01 etter.mdns.examples
-rw-r--r-- 1 root root 2,3K jul  4 12:01 etter.mime
-rw-r--r-- 1 root root 1,7K jul  4 12:01 etter.nbns
-rw-r--r-- 1 root root 104K jul  4 12:01 etter.services
-rw-r--r-- 1 root root 2,8K jul  4 12:01 etter.ssl.crt
-rw-r--r-- 1 root root 5,9K jul  4 12:01 Makefile
root@Kali:/home/tiago/ettercap/build#

--- a/share/CMakeLists.txt
+++ b/share/CMakeLists.txt
@@ -7,6 +7,7 @@ endif()
 
 set(EC_CONFFILES
     etter.dns
+    etter.mdns
     etter.nbns
 )
 
@@ -42,8 +43,6 @@ endforeach()
 install(FILES
         ${CMAKE_CURRENT_BINARY_DIR}/${EC_CONFFILES}
         ${CMAKE_CURRENT_BINARY_DIR}/etter.conf
-        ${CMAKE_CURRENT_BINARY_DIR}/etter.dns
-        ${CMAKE_CURRENT_BINARY_DIR}/etter.mdns
         DESTINATION ${INSTALL_SYSCONFDIR}/ettercap)
 install(FILES
         ${CMAKE_CURRENT_BINARY_DIR}/${EC_DATAFILES}

